share/lib/python/neuron/__init__.py

```diff
@@ -143,24 +143,46 @@
 __version__ = version
 _original_hoc_file = None
 if not hasattr(hoc, "__file__"):
-  import platform
-  import os
-  p = h.nrnversion(6)
-  if "--prefix=" in p:
-    p = p[p.find('--prefix=') + 9:]
-    p = p[:p.find("'")]
+  # first try is to derive from neuron.__file__
+  p = None
+  if sys.version_info[0] == 2:
+    # python 2 seems to have hoc.__file__ already filled in so never get here.
+    try:
+      import imp
+      mspec = imp.find_module("neuron")
+      p = mspec[1]
```

Comment:
just a suggestion / nitpick : could we use longer variable name? `p` is here install prefix?

Reply:
Will change to "origin"
```origin = None # path to neuron/__init__.py```